CHANGELOG.md

```diff
@@ -134,10 +134,17 @@ While some features like the clipboard, menus or file dialogs are not yet availa
 [#878]: https://github.com/xi-editor/druid/pull/878
 [#889]: https://github.com/xi-editor/druid/pull/899
 
-## [0.5] - 2020-04-01
+## [0.5.0] - 2020-04-01
 
 Last release without a changelog :(
 
+## [0.4.0] - 2019-12-28
+## [0.3.2] - 2019-11-05
+## [0.3.1] - 2019-11-04
```

Comment:
Shouldn't this be 2019-11-05 as well? :thinking:

Reply:
@xStrom I'm not sure what you mean, it seems correct to me.

Reply:
Haha, well if it's not a typo then both GitHub and crates.io show the date in my timezone. That's why I see 2019-11-05.